README.md

```diff
@@ -78,9 +78,13 @@ server that is viewable with Kibana and Grafana!
 
 Look at some of the other benchmarks for examples of how this works.
 
+Note: Please tag @aakarshg for review when adding a new benchmark wrapper especially ones that index data into elasticsearch and mention:
+* which field(key) in the document will be of the type date(time)
+* highly recommend that the key of the type date(time) use epoch format (helps a lot when dealing with tool run in multiple timezones like snafu)
```

Comment:
Which benchmark does this today?   If none, why don't we start by converting some benchmark to use this if we haven't already, and verify that it works, before we make this recommendation?  I volunteer to use fs-drift for this, because it's not hard to change and won't impact at lot of people so far.   If this one works then I can help with the other ones.

Reply:
This is how Scribe is indexing today. I just shared with @dry923 how our templates need to be on the ES side.

Reply:
Where is the Scribe code that does it?  Maybe we could make a short example illustrating how to do it from this code.

Reply:
upto the user but it can be done as follows: `int(time.time())`